# This message tells with buttons are pressed on the CM730

bool button1
bool button2
bool button3

